tt_eager/tt_dnn/op_library/composite/composite_ops.cpp

```diff
@@ -900,8 +909,24 @@ Tensor _div(const Tensor& input_a, const Tensor& input_b, bool accurate_mode, co
         result,
         output_mem_config);
 }
-Tensor div(const Tensor& input_a, const Tensor& input_b, bool accurate_mode, const MemoryConfig& output_mem_config) {
-    return operation::decorate_as_composite(__func__, _div)(input_a, input_b, accurate_mode, output_mem_config);
+Tensor div(const Tensor& input_a, const Tensor& input_b, bool accurate_mode, string round_mode, const MemoryConfig& output_mem_config) {
+    return operation::decorate_as_composite(__func__, _div)(input_a, input_b, accurate_mode, round_mode, output_mem_config);
+}
+
+Tensor _div_overload(const Tensor& input_a, float scalar, bool accurate_mode, string round_mode,  const MemoryConfig& output_mem_config) {
+    Tensor result = div_unary(input_a, scalar);
+
+    if(round_mode == "trunc"){
+        result = trunc(result);
+    }
+    else if(round_mode == "floor"){
+        result = floor(result);
+    }
```

Comment:
Should assert out for unsupported/unknown round mode

Reply:
Sure, will add the assert statement